editor/plugins/spatial_editor_plugin.cpp

```diff
@@ -5674,6 +5682,32 @@ void SpatialEditor::_init_grid() {
 		grid_mat[c]->set_shader_param("grid_size", grid_fade_size);
 		grid_mat[c]->set_shader_param("orthogonal", orthogonal);
 
+		// cache these so we don't have to re-access memory.
```

Comment:
```suggestion
		// Cache these so we don't have to re-access memory.
```
https://docs.godotengine.org/en/stable/community/contributing/code_style_guidelines.html#comment-style-guide

Reply:
Fixed

Reply:
... doesn't look fixed yet 🙂